File: output/CodeLlama/codenet/C++/Python/s605014348.py
import math

def div(n):
    if n % 2 == 0:
        return 2
    s = math.sqrt(n)
    for i in range(3, int(s) + 1, 2):
        if n % i == 0:
            return i
    return n

num = int(input())
print(num, ":", end="")
while num != 1:
    print(" " + str(div(num)), end="")
    num = num // div(num)
print()

File: output/ChatGPT/codenet/C++/Java/s605014348.java
import java.util.Scanner;

public class s605014348 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        System.out.print(num + ":");

        while (num != 1) {
            System.out.print(" " + div(num));
            num = num / div(num);
        }
        System.out.println();
    }

    public static int div(int n) {
        if (n % 2 == 0) return 2;
        int s = (int) Math.sqrt(n);
        for (int i = 3; i <= s; i += 2) {
            if (n % i == 0) return i;
        }

        return n;
    }
}